src/ray/core_worker/core_worker.cc

```diff
@@ -1556,6 +1556,14 @@ Status CoreWorker::Get(const std::vector<ObjectID> &ids,
   for (const ObjectID &id : ids) {
```

Comment:
Done, though I pass a single object ID rather than a vector. The loop has logic that is specific to CoreWorker (all objects must be experimental or all objects must not be), so I thought it was best to leave it in CoreWorker.